src/styles/blocks/body.scss

```diff
@@ -0,0 +1,6 @@
+@use "../utils/functions";
+.body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: $font-weight-normal;
```

Comment:
The variable `$font-weight-normal` is used here, but it is not defined within this file or imported from another file. Ensure that this variable is defined in the `functions` file or any other imported file, or replace it with a valid font-weight value.